### PR TITLE
Remove deprecated service version

### PR DESCRIPTION
**Pull request by timotheeguerin**

### CODE REVIEW

File: packages/cadl-ranch-specs/http/client/structure/common/service.tsp [hunk 1]
```diff
@@ -29,7 +29,6 @@ using Azure.ClientGenerator.Core;
 )
 @service({
   title: "MultiClient",
-  version: "1.0.0",
```
Comment (qiaozha) [inline packages/cadl-ranch-specs/http/client/structure/common/service.tsp:32]: we should add `@versioned` to avoid breaking ?

Comment (timotheeguerin) [reply packages/cadl-ranch-specs/http/client/structure/common/service.tsp:32]: this should not have been used and is the entire point why we deprecating `version:`

Comment (qiaozha) [reply packages/cadl-ranch-specs/http/client/structure/common/service.tsp:32]: oh because there's no api version parameter in it? got it, thanks

**Review by qiaozha — APPROVED** [review]

**Review by weidongxu-microsoft — APPROVED** [review]

File: packages/cadl-ranch-specs/http/client/structure/common/service.tsp (hunk 1 above)
Comment (timotheeguerin) [reply packages/cadl-ranch-specs/http/client/structure/common/service.tsp:32]: yeah especially but either way

**Pull request merged**